### DISCUSSION TITLE
What is the current stable version of Meshcentral

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#6063] [Q&A]
Hi Community , 
                   I want to host a Meshcentral server as a public server for my personal use. So what the stable version of Meshcentral? I may make changes or modifications in source code in the future as needed. is it 1.1.0 ?

**Comment by PathfinderNetworks** [comment]
The Check Server Version app shows it as being 1.1.0.  But that's pretty outdated, in my honest opinion. I always run the most current release (which is currently 1.1.22).  Absolutely no issues at all with that most current version.

**Comment by amoljagdalepucsd** [comment]
Okay Thanks @PathfinderNetworks.
@si458  can i host latest one ?

**Reply by si458** [reply]
Anyone can host the latest,
just `npm install meshcentral` or use the `latest` docker image
